stores/authStore.ts

```diff
@@ -12,5 +12,7 @@ export const useAuthStore = defineStore('auth', {
   },
   getters: {
     isLoggedIn: (state) => !!state.user,
+    hasAccessLevel: (state) => (accessLevel?: string) =>
+      accessLevel ? state.user?.accessLevel?.includes(accessLevel) : true,
```

Comment:
LGTM. Maybe go through this syntax with the other group members at some point? Or leave a comment. The `?` syntax might look a bit scary coming from Python :)